src/duckling/game/collision/collision-attribute.ts

```diff
@@ -1,26 +1,51 @@
 import {Attribute, Entity} from '../../entitysystem/entity';
 import {Vector} from '../../math/vector';
 import {Box2} from '../../math/box2';
+import {SelectOption} from '../../controls';
 
 export const COLLISION_KEY = "collision";
 
 /**
  * Describes how the entity behaves in the solid body system.
  */
-export enum BodyType {
-    None,
-    Solid,
-    Environment
-}
+export type BodyType = "none" | "solid" | "environment";
```

Comment:
I'm repeating all the values here to create the Type and also create the SelectOptions used in the array-select component. I don't *think* there's a way to reuse these values in one situation, but it's worth a bit more investigation. Or possibly just ditch the super strict type safety.